pkg/proxy/ipvs/proxier_test.go

```diff
@@ -4835,6 +4835,183 @@ func TestTestInternalTrafficPolicyE2E(t *testing.T) {
 	}
 }
 
+// Test_EndpointSliceReadyAndTerminatingCluster tests that when there are ready and ready + terminating
+// endpoints and the traffic policy is "Cluster", only the ready endpoints are used.
+func Test_EndpointSliceReadyAndTerminatingCluster(t *testing.T) {
```

Comment:
would be good to get someone familiar with the IPVS unit tests to look at this...